src/Service.Tests/Authorization/GraphQL/GraphQLMutationAuthorizationTests.cs

```diff
@@ -110,8 +115,25 @@ private static SqlMutationEngine SetupTestFixture(bool isAuthorized)
             Mock<IQueryBuilder> _queryBuilder = new();
             Mock<IHttpContextAccessor> httpContextAccessor = new();
             Mock<ILogger<SqlMutationEngine>> _mutationEngineLogger = new();
+            MockFileSystem fileSystem = new();
+            fileSystem.AddFile(FileSystemRuntimeConfigLoader.DEFAULT_CONFIG_FILE_NAME, new MockFileData(string.Empty));
+            FileSystemRuntimeConfigLoader loader = new(fileSystem);
+            RuntimeConfigProvider provider = new(loader);
             DefaultHttpContext context = new();
-            Mock<GQLFilterParser> _gQLFilterParser = new(_sqlMetadataProvider.Object);
+            Mock<IMetadataProviderFactory> _metadataProviderFactory = new();
+
+            _metadataProviderFactory.Setup(x => x.GetMetadataProvider(It.IsAny<string>())).Returns(_sqlMetadataProvider.Object);
```

Comment:
Should `x.GetMetadataProvider(It.IsAny<string>())` be `x.GetMetadataProvider(string.Empty)` (or "") to simulate realistic Single database usage? If i remember correctly, your metadataprovider code will only return the default database if the provided datasourceName is empty (and/or null?)

Reply:
or maybe you can grab the actual default single database source name and use that. That would help with backwards compat to ensure that existing code doesn't try to fetch a non-existing databasesource.